deploy/tests/test_update_drones_etryvoga_v1.py

```
@@ -55,73 +60,140 @@ async def test_2(mock_get_etryvoga, mock_get_cache_data):
-            "version": 1,
-            "states": {str(region["id"]): {"lastUpdate": "2025-01-15T10:00:00Z"}},
-            "info": {
-                "last_update": "2025-01-26T19:18:55Z",
-                "last_id": "239a016a03c583633424afb5d418051b0a33a59374d0884912f8062336c09a93",
+    for _, region in regions.items():
+
+        def mock_get_cache_data_side_effect(mc, key, default=None):
+            mock_responses = {
+                b"drones_etryvoga": {
+                    "version": 1,
+                    "states": {str(region["id"]): {"lastUpdate": "2025-01-15T10:00:00Z"}},
```

Ruff (0.8.2)

Function definition does not bind loop variable `region`

(B023)